### ISSUE TITLE
Change proposal

### ISSUE THREAD

**Issue by klodoma** [garris/BackstopJS#1349]
In the current situation the `html_report` contains always the latest report and the older reports are lost. 

I propose to add a "flag" where the `html_report` folder gets a similar structure to the `bitmaps_test` folder (with timestamps as sub-folders) which contains then the actual reports. 

This helps if one executes the test repeatedly and want to see the diffs. 

Any thoughts against such a feature?

![Screenshot_6](https://user-images.githubusercontent.com/944350/128719885-ddcbb413-5c39-438e-af44-b8d6171e2af3.png)

**Comment by garris** [comment]
@klodoma yes, I am supportive.  Others have asked for this as well.  It could be as simple as an operation which runs after the current reporting flow completes and copies all dependent assets into an additional html_report directory structure. 

SideNote:  I personally don't have a big problem with redundant assets floating around to ensure backwards compatibility -- so pure copy functions are fine in this solution are with _me_ -- but those concerns sometimes come up for others.  That said, I would approach any concerns of cleaning up old/redundant files as a separate housecleaning problem.

**Comment by klodoma** [comment]
@garris I agree with redundancy, this is not something one will always use. 

We could add a flag like: `keepReports: true` or `reportsHistory: true`, I would go with the former.

**Comment by garris** [comment]
I like `keepReports` -- but maybe `archiveReports` is a little more accurate (cause we would literally be creating self-contained report archives.)

**Comment by nicolasgasco** [comment]
Hello, everyone! Is this feature still available? We're struggling with generating an HTML report that includes all timestamped directories that are generated and not simply the latest one.
We tried activating the flag introduced in the MR attached to this issue, but to no avail.
